bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -711,6 +712,26 @@ object TimeProblems {
   }
 }
 
+case class Semester2020AProblems(p: Proposal, s: ShellAdvisor) {
```

Comment:
i don't recall this very well but do we need to keep `SemesterXXXXProblems` of previous semesters?

Reply:
I introduced the `SemesterXXXXXProblems` just to segregate things to make them easier to know when they were introduced, IIRC, but you are right: looking at them, we do not need the older ones. I'll go ahead and remove them.

Reply:
I got rid of the two outdated `SemesterXXXXXProblems` and graduated `dssiObsolete` to a permanent problem.